package.json

```diff
@@ -26,18 +28,18 @@
     "@typescript-eslint/parser": "^5.47.1",
     "eslint": "^8.31.0",
     "jest": "^29.3.1",
-    "ngrok": "^4.3.3",
+    "ngrok": "^5.0.0-beta.2",
```

Comment:
Old versions are no longer supported to run with free account thus we had to upgrade to latest

Reply:
This seems to be a major upgrade of package
What are the steps you have ensured for complete functionality of application being working as expected

Reply:
beta can be unstable for some features please check @RitikJaiswal75

Reply:
hey @harshith-venkatesh the following package is an external dependency it does not hinder the current functionality of the application for users, it is a dev dependency and improves the developer experience

Hey @shreya-mishra nice of you to point us using beta, since this is only used for port forwarding in our feature I have verified it working